network/src/retry.rs

```diff
@@ -44,7 +44,7 @@ impl RetryConfig {
     // default gives 5-6 retries in ~30 s total retry time.
 
     /// Default for [`RetryConfig::max_retry_interval`] (500 ms).
-    pub const DEFAULT_INITIAL_RETRY_INTERVAL: Duration = Duration::from_millis(100);
+    pub const DEFAULT_INITIAL_RETRY_INTERVAL: Duration = Duration::from_millis(500);
 
     /// Default for [`RetryConfig::max_retry_interval`] (15 s).
     pub const DEFAULT_MAX_RETRY_INTERVAL: Duration = Duration::from_secs(1);
```

Comment:
While we're here do we want to bump this up too? Maybe to match the comment?

Reply:
@bmwill done!